build.gradle.kts

```diff
@@ -136,7 +136,7 @@ dependencies {
 val lintPaths = listOf(
     "smithy-kotlin-codegen/src/**/*.kt",
     "runtime/**/*.kt",
-    "benchmarks/**/jvm/*.kt",
+    "tests/**/**/jvm/*.kt",
```

Comment:
**Comment**: Looks like this was incorrect before and it's still incorrect now. We don't need an extra `**` before `jvm` (because `**/**` is equivalent to `**`) but we _do_ need a `**` between `jvm` and `*.kt`:

```
"tests/**/jvm/**/*.kt",
```

Changing this picks up several new problems in **HttpEngineBenchmarks** and some existing ones in **AwsSignerBenchmark**.

Reply:
How embarrassing, good catch I'll fix it up.